lmdk/cmake/ldscripts/bss_linker_script.txt

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Can we have a comment in each linker script about what it does and how it's used. i.e. module developers should be able to figure out the scripts used for thier use case and integrate/debug features as needed.

Reply:
Added description about linker scripts to ldscripts.cmake.